apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,26 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2019 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "mbed.h"  // header file for microcontroller library
+#include "CANMsg.h"
+#include "TutorialServo.h"
+
+// declare global objects
+CAN can_rx(D4, D10);
```

Comment:
FW uses a custom `CANBus` wrapper class instead of mbed's `CAN` API directly. Not sure if it's required for the training, but it's probably best practice to use that instead. See the arm app for an example